server/rooms.ts

```diff
@@ -177,44 +184,54 @@ export abstract class BasicRoom {
 
 		// room settings
 
-		this.chatRoomData = null;
-		this.isPrivate = false;
+		this.settings = {
+			aliases: [],
+			auth: null,
```

Comment:
```suggestion
			auth: {},
```

Reply:
Huh, i thought i had committed that. I know i did it locally. Apparently not?